features/graph/documentation/README.md

```diff
@@ -0,0 +1,36 @@
+# Netlify Graph
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Components](#components)
+- [Enabling the feature](#enabling-the-feature)
+- [Feedback](#feedback)
+
+## Overview
```

Comment:
[question] @dend, has there been previous discussion about mentioning OneGraph in relation to Netlify Graph? I think mentioning that Netlify Graph is built on OneGraph could bolster the feature's attractiveness. However, I could also see an argument for not mentioning OneGraph - letting Netlify Graph build its own reputation.

Reply:
@KyleBlankRollins in the docs, let's make sure to just refer to Netlify Graph. There might be an opportunity for us to chat more about the OneGraph aspect once we solidify the plans/get this ready for GA.